attest/verifier/config/README.md

```diff
@@ -5,13 +5,11 @@ A JSON schema for basic attestation configs.
 
 This crate defines a schema for storing a set of trusted measurements for SGX
 enclaves in json. These measurements can be grouped by version, and named.
```

Comment:
I wasn't sure how much to modify this readme.
I think the higher level JSON format is still desired, currently it's the types contained in the `TrustedMeasurementSet` that is changing.
Later on we may need to update the advisories to be something like
```json
advisories: {
  ids: [],
  status: "SWHardeningNeeded",
}

Reply:
no longer done this way. Moving back to sw and config advisories